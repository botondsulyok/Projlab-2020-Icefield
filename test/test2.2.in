loadmap testmap2
stats f1
stats f2
stats f3