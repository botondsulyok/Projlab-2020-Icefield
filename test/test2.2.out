id: f1
type: unstable
building: plains0
snow: 3
pickable: null
standers: null
pullers: null
masters: null
neighbours: f2, f3
capacity: 5
broken: false

id: f2
type: unstable
building: plains1
snow: 2
pickable: null
standers: null
pullers: null
masters: null
neighbours: f1
capacity: 6
broken: false

id: f3
type: iceunit
building: plains2
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: f1

